deployment/installer/opik_installer/reporting.py

```diff
@@ -0,0 +1,74 @@
+import os
+import requests
+import platform
+from opik_installer import version
+
+def _add_event_metadata(event):
+    event["event_properties"]["opik_installer_version"] = version.__version__
+    event["event_properties"]["os"] = platform.system()
+    event["event_properties"]["os_version"] = platform.version()
+    event["event_properties"]["architecture"] = platform.machine()
+    
+    return event
+
+def _create_install_event(success: bool, failure_reason: str = None):
+    event = {
+        "user_id": "Anonymous",
+        "event_type": "opik_installer_install",
+        "event_properties": {
+            "successful": success,
```

Comment:
I would add OS type and opik version to the events

Reply:
Both are added through the _add_event_metadata function

Reply:
We have:
    event["event_properties"]["opik_installer_version"] = version.__version__
    event["event_properties"]["os"] = platform.system()
    event["event_properties"]["os_version"] = platform.version()
    event["event_properties"]["architecture"] = platform.machine()

Reply:
worth getting python version as well ?
```sys.version.split()[0]```